### PR TITLE
Cw/wrap fixes

### PR DESCRIPTION
**Pull request by chriswynn**
Make wrap scale in a more fluid way

### CODE REVIEW

File: src/components/Wrap/index.tsx [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
-    css({
-      marginLeft: "auto",
-      marginRight: "auto",
-      maxWidth: size && theme.maxWidths[size - 1],
-      width: "100%",
-    });
-
-  return <W css={styles} {...props} />;
+  return (
+    <W
```
Comment (sethkrasnianski) [inline src/components/Wrap/index.tsx]: I know this was added previously, but reading this from a specification standpoint, I can't seem to draw significance from the parameter `W`. Perhaps `El` would better convey its purpose?

File: src/components/Wrap/index.tsx [hunk 2]
```diff
@@ -1,25 +1,32 @@
 /** @jsx jsx */
 import * as React from "react";
-import { jsx, css } from "@emotion/core";
+import { jsx } from "@emotion/core";
+import { mq } from "../../../lib/utils";
```
Comment (sethkrasnianski) [inline src/components/Wrap/index.tsx]: What was the purpose behind storing `lib` outside of the `src` directory?

File: src/components/Wrap/index.tsx (hunk 1 above)
Comment (chriswynn) [reply src/components/Wrap/index.tsx]: Yeah `El` makes a lot more sense

File: src/components/Wrap/index.tsx (hunk 2 above)
Comment (chriswynn) [reply src/components/Wrap/index.tsx]: I think it was there to begin with so I just continued using it. Moving into src would make more sense

File: src/components/Wrap/index.tsx (hunk 1 above)
Comment (chriswynn) [reply src/components/Wrap/index.tsx]: This will be fixed with #33

File: src/components/Wrap/index.tsx (hunk 2 above)
Comment (chriswynn) [reply src/components/Wrap/index.tsx]: This will be fixed with #33

File: src/components/Column/index.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { Theme } from "../../types/global";
 const Column: React.FC = props => (
   <div
     css={({ ...theme }: Theme) => ({
-      flexBasis: "calc((60rem - 100%) * 9999)",
+      flexBasis: "calc((42rem - 100%) * 9999)",
```
Comment (jssee) [inline src/components/Column/index.tsx]: should we codify this calc in the theme?

**Pull request merged**